index.js

```diff
@@ -23,8 +24,8 @@ async function startServer() {
     app.use(express.urlencoded({ extended: false }));
     app.use(cors());
     app.use(session());
-    //TODO: Add trim body
+    app.use(trimBoddy());
```

Comment:
Is this really needed? Shouldn't this delay too much the requests as it must loop through all keys, to trim them and etc...?